docs/docs/platform/quickstarts/redwoodjs.mdx

```diff
@@ -0,0 +1,1178 @@
+              No account yet?{' '}
+              <Link
+                to={routes.signUp()}
+                className="text-blue-600 hover:text-blue-500 hover:underline hover:underline-offset-1 transition"
+              >
+                Sign up
+              </Link>
+            </p>
+          ) : null}
+        </div>
+      </div>
+    </>
+  )
+}
+
+export default SignInPage
+```
+
+![RedwoodJS - Sign-in page](/img/quickstarts/redwoodjs-sign-in-page.png)
+
+#### 3. Sign-out
+
+Finally, to allow the users to sign out from the app, we can use the RedwoodJS
+`logOut` method from the `useAuth` hook:
+
+```jsx title="web/src/layouts/DashLayout/DashLayout.js"
+// highlight-next-line
+import { useAuth } from '@redwoodjs/auth'
+
+const DashLayout = ({ children }) => {
+  // highlight-next-line
+  const { logOut } = useAuth()
+
+  const menuItems = [
+    //...
+    {
+      label: 'Logout',
+      // highlight-next-line
+      onClick: logOut,
+      icon: LogoutIcon
+    }
+  ]
+
+  //...
+}
+```
+
+### Protect routes
+
+Now that we have implemented authentication, we can easily decide who can access
+certain parts of our application.
+
+In our case, we'll only allow authenticated users to have access to the `/` and
+`/profile` routes. All the other users should be redirected to the `/sign-in`
+page if they try to access those routes.
+
+To do so, we can wrap those two routes inside the `Private` component provided by the built-in RedwoodJS router:
+
+```jsx title="web/src/Routes.js"
+// highlight-next-line
+import { Private, Router, Route, Set } from '@redwoodjs/router'
+import DashLayout from './layouts/DashLayout/DashLayout'
+
+const Routes = () => {
+  return (
+    <Router>
+      // highlight-next-line
+      <Private unauthenticated="signIn">
+        <Set wrap={DashLayout}>
+          <Route path="/" page={HomePage} name="home" />
+          <Route path="/profile" page={ProfilePage} name="profile" />
+        </Set>
+        // highlight-next-line
+      </Private>
+      <Route path="/sign-up" page={SignUpPage} name="signUp" />
+      <Route path="/sign-in" page={SignInPage} name="signIn" />
+      <Route notfound page={NotFoundPage} />
+    </Router>
+  )
+}
+```
+
+### Retrieve user data
+
+Finally, let's display the information of the authenticated user throughout his
+dashboard to make the app more personalized.
+
+Getting the current authenticated user data is quite easy. Indeed, we
+can use the `userMetadata` variable returned by the `useAuth` hook.
+
+So, open the files where we need the user's data and retrieve it like so:
+
+<Tabs groupId="files">
+  <TabItem value="DashLayout" label="DashLayout" default>
+
+```js title="web/src/layouts/DashLayout/DashLayout.js"
+const DashLayout = ({ children }) => {
+  // highlight-next-line
+  const { logOut, userMetadata: user } = useAuth()
+  //...
+}
+```
+
+  </TabItem>
+  <TabItem value="HomePage" label="HomePage">
+
+```jsx title="web/src/pages/HomePage/HomePage.js"
```

Comment:
The default HomePage.js from RedwoodJS looks a bit different.


```js
import { Link, routes } from '@redwoodjs/router'
import { MetaTags } from '@redwoodjs/web'

const HomePage = () => {
  return (
    <>
      <MetaTags title="Home" description="Home page" />

      <h1>HomePage</h1>
      <p>
        Find me in <code>./web/src/pages/HomePage/HomePage.js</code>
      </p>
      <p>
        My default route is named <code>home</code>, link to me with `
        <Link to={routes.home()}>Home</Link>`
      </p>
    </>
  )
}

export default HomePage
```

I think the code here should be close to what's in the default HomePage.js file

Maybe something like this: 

```js
import { Link, routes } from '@redwoodjs/router'
import { MetaTags } from '@redwoodjs/web'
import { useAuth } from '@redwoodjs/auth'

const HomePage = () => {
  const { userMetadata: user } = useAuth()

  return (
    <>
      <MetaTags title="Home" description="Home page" />

      <h1 className="text-3xl font-semibold">HomePage</h1>

      <p className="mt-2 text-lg">
        Welcome, {user?.metadata?.firstName || 'stranger'}{' '}
        <span role="img" alt="hello">
          👋
        </span>
      </p>

      <p className="mt-4 text-gray-500">
        Find me in <code>./web/src/pages/HomePage/HomePage.js</code>
      </p>
      <p className="mt-4 text-gray-500">
        My default route is named <code>home</code>, link to me with `
        <Link to={routes.home()}>Home</Link>`
      </p>
    </>
  )
}

export default HomePage
```

Reply:
✅ Updated.